---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

# Yuzhou Chang
---
- Department of Biomedical Informatics, College of Medicine, The Ohio State University
- Pelotonia Institute for Immuno-Oncology
- Center for Virology and Vaccine Research, Beth Israel Deaconess Medical Center, Harvard Medical School

2255 Kenny Rd, Columbus, OH 43210  
**Email**: yuzhou.chang@osumc.edu | chang.1754@osu.edu | ychang13@bidmc.harvard.edu  
**GitHub**: [BMEngineeR](https://github.com/BMEngineeR)  

[CV download](https://github.com/BMEngineeR/yuzhouchang.github.io/blob/master/files/CV-Yuzhou%20Chang.pdf)
---

## Education

- **Postdoc** (01/2024 – Present)  
  Joint Postdoctoral Researcher  
  - Immuno-Oncology Informatics Group, Pelotonia Institute for Immuno-Oncology, The Ohio State University  
  - Beth Israel Deaconess Medical Center, Harvard Medical School  

- **Ph.D.** (08/2018 – 12/2023)  
  Biomedical Sciences Graduate Program, The Ohio State University  
  - Advisors: Prof. Qin Ma, Prof. Zihai Li  
  - Training: Statistical modeling, bioinformatics, computational system biology  

- **B.E.** (09/2012 – 06/2016)  
  Biopharmaceutical, China Pharmaceutical University  
  - Advisor: Weihua Chu  

---

## Honors & Awards

- **ICIBM 2023 Student & Trainee Travel Award** (07/2023)  
- **Award for Excellence in Research (Graduate Student)** (12/2021)  
- **Outstanding Student Leader** (3 times: 09/2013–09/2015)  
- **Second-Class Scholarship** (09/2015)  

---

## Research Interests

- Graph signal processing for spatial transcriptomics.  
- Developing computational tools for spatial omics data analysis, including cell-cell communication and tissue architecture identification based on deep learning.  
- Applying spatial transcriptomics to immuno-oncology.  
- Gene regulatory network inference.  

---

## Tool Development

- **SpaGFT**: Spatial omics feature representation based on Graph Fourier Transform ([doc](https://spagft.readthedocs.io/en/latest/index.html)).  
- **RESEPT**: A deep learning framework for tissue architecture identification based on spatial transcriptomics data ([GitHub](https://github.com/OSU-BMBL/RESEPT)).  
- **SPRUCE**: Bayesian multivariate mixture model for spatial transcriptomics data ([GitHub](https://github.com/carter-allen/spruce)).  
- **IRIS-FGM**: An integrative single-cell RNA-Seq interpretation system for functional module analysis ([Bioconductor](https://bioconductor.org/packages/release/bioc/html/IRISFGM.html)).  
- **scGNN**: Single-cell Graph Neural Network ([GitHub](https://github.com/juexinwang/scGNN)).  

---

For a full list of publications, visit [Google Scholar](https://scholar.google.com/citations?user=yi3ClTQAAAAJ&hl=en).  
